lib/css-declaration-sorter.js

```diff
@@ -42,18 +44,27 @@ module.exports = {
   },
   config: {
     formatOnSave: {
+      order: 1,
       title: 'Format file on save',
       description: 'Automatically format file when saving.',
       type: 'boolean',
       default: false,
     },
     sortOrder: {
+      order: 2,
       title: 'Default sorting order',
       description: 'Select the default sorting order.',
       type: 'string',
       default: 'alphabetical',
-      enum: ['alphabetical', 'smacss', 'concentric-css'],
+      enum: ['alphabetical', 'smacss', 'concentric-css', 'custom'],
     },
+    customOrder: {
+      order: 3,
+      title: 'Custom sorting order',
+      description: 'Select the default sorting order.',
+      type: 'string',
+      default: 'place your own order',
```

Comment:
```suggestion
      default: '',
```
Maybe just leave it empty? Or have a tiny example like `z-index border`